### PR TITLE
ceph.spec.in: disable system_pmdk on s390x for SUSE distros

### PR DESCRIPTION
**Pull request by tserong**
Fixes: https://tracker.ceph.com/issues/57860
Signed-off-by: Tim Serong <tserong@suse.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by tserong** [comment]
jenkins test api

**Comment by tserong** [comment]
jenkins test make check

File: ceph.spec.in [hunk 1]
```diff
@@ -57,7 +57,11 @@
 %global _remote_tarball_prefix https://download.ceph.com/tarballs/
 %endif
 %if 0%{?suse_version}
+%ifarch s390x
```
Comment (idryomov) [inline ceph.spec.in:60]: Did you see that it was also disabled for aarch64 in a followup -- https://github.com/ceph/ceph/pull/47544.  It looks like SUSE doesn't need that but I wanted to double check.

Comment (tserong) [reply ceph.spec.in:60]: Good catch :-) but it's OK, SUSE does have an aarch64 build of pmdk.

**Review by idryomov — APPROVED** [review]

**Comment by tserong** [comment]
Built fine on shaman for CentOS 8 x86_64 (https://shaman.ceph.com/builds/ceph/wip-tserong-testing/6c2fd4267e58abdef2f1ffcd8a929a479c06c3cc/default/320945/) and CentOS 9 x86_64 (https://shaman.ceph.com/builds/ceph/wip-tserong-testing/6c2fd4267e58abdef2f1ffcd8a929a479c06c3cc/default/320942/).

Tested internally at SUSE against SLE (no public link to build available).

Is that sufficient QA for this PR?

**Comment by idryomov** [comment]
> Is that sufficient QA for this PR?

I think so!

**Pull request merged**